src/types/devnet.ts

```diff
@@ -34,6 +36,36 @@ export interface Devnet {
         networkId?: string
     ) => Promise<LoadL1MessagingContractResponse>;
 
+    /**
+     * Sends a mock tx from L1 to L2 without running L1.
```

Comment:
Is it a mock tx or a mock message?

Reply:
https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/294/commits/17f0f05a139efaa1c00fdfa4ded82c6d2dd51480